js/parser.js

```diff
@@ -0,0 +1,24 @@
+
+/*
+var usersOnSite = document.getElementById("user-site");
+*/
+var users = 'https://ubyssey-analytics.appspot.com/query?id=ahNzfnVieXNzZXktYW5hbHl0aWNzchULEghBcGlRdWVyeRiAgICAgICACgw&format=json';
+var $usersite = $('#usersite');
+
+$(function() {
+$.ajax({
+    type: 'GET',
+    url: 'usersperhour.json',
+        xhrFields: {
```

Comment:
So if we remove the xhrFields thing and add ```dataType: 'jsonp'``` this should be able to fetch the data from the SuperProxy.